deployment.toml

```diff
@@ -1,27 +1,32 @@
-# [[cells]]
-# name = "my_cell"
-# enable_type_id = false
-# location = { file = "build/release/my_cell" }
-#
-# # reference to on-chain cells
-# [[cells]]
-# name = "genesis_cell"
-# enable_type_id = false
-# location = { tx_hash = "0x71a7ba8fc96349fea0ed3a5c47992e3b4084b031a42264a018e0072e8172e46c", index = 0 }
- 
+[[cells]]
+name = "ckb-cheque-script"
+enable_type_id = true
+location = { file = "build/release/ckb-cheque-script" }
 
-# # Dep group cells
-# [[dep_groups]]
-# name = "my_dep_group"
-# cells = [
-#   "my_cell",
-#   "genesis_cell"
-# ]
+# reference to on-chain cells
+[[cells]]
+name = "secp256k1_data"
+enable_type_id = false
+location = { tx_hash = "0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37", index = 3 } 
 
-# # Replace with your own lock if you want to unlock deployed cells.
-# # For example the secp256k1 lock
-# [lock]
-# code_hash = "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"
-# args = "0x0000000000000000000000000000000000000000"
-# hash_type = "type"
+[[cells]]
+name = "secp256k1_blake2b_sighash_all_dual"
+enable_type_id = false
+location = { file = "ckb-miscellaneous-scripts/build/secp256k1_blake2b_sighash_all_dual" }
+
+# Dep group cells
+[[dep_groups]]
+name = "dep_group"
+cells = [
+  "ckb-cheque-script",
+  "secp256k1_data",
+  "secp256k1_blake2b_sighash_all_dual"
+]
+
+# Replace with your own lock if you want to unlock deployed cells.
+# For example the secp256k1 lock
+[lock]
+code_hash = "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"
+args = "0x0000000000000000000000000000000000000000"
```

Comment:
Can it be upgraded later if args is set to 0?

Reply:
No. The `args` is the `lock args` of the deployer.

Reply:
No. The deployer should update with his own `lock args` and I deployed the script with my lock args.

Reply:
oh, this just an example.

Reply:
Maybe I can give a description here.